spec/cypress/end_to_end/admin.cy.js

```diff
@@ -0,0 +1,30 @@
+describe('Admin feature tests', () => {
```

Comment:
I ran this test locally with `yarn cypress run --project spec --spec "spec/cypress/end_to_end/admin.cy.js"` and confirm that it passes.